dipy/core/sphere.py

```diff
@@ -496,6 +497,44 @@ def disperse_charges(hemi, iters, const=.2):
     return HemiSphere(xyz=charges), potential
 
 
+def fibonacci_sphere(n_points=724, randomize=True):
+    """
+    Generate points on the surface of a sphere using Fibonacci Spiral Sampling.
+
+    Parameters
+    ----------
+
+    - n_points (int): The number of points to generate on the sphere surface.
+      Default is 724.
+    - randomize (bool): If True, randomize the starting point on the sphere.
+      Default is True.
```

Comment:
Can you remove the default value. 

Also, Can you look at the other function to respect our docstring standard?  currently your docstring does not follow it